social_protection/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from core import models as core_models
+from policyholder.models import PolicyHolder
+
+
+class BenefitPlan(core_models.HistoryBusinessModel):
+    code = models.CharField(db_column='Code', max_length=8, null=False)
+    name = models.CharField(db_column='NameBF', max_length=255, null=False)
+    date_from = models.DateTimeField(db_column="DateFrom")
+    date_to = models.DateTimeField(db_column="DateTo")
+    max_beneficiaries = models.SmallIntegerField(db_column="MaxNoBeneficiaries")
+    ceiling_per_beneficiary = models.DecimalField(db_column="BeneficiaryCeiling",
+                                                  max_digits=18,
+                                                  decimal_places=2,
+                                                  blank=True,
+                                                  null=True,
+                                                  )
+    organization = models.ForeignKey(PolicyHolder, models.DO_NOTHING, db_column='Organization', blank=True, null=True)
+    schema = models.TextField(db_column="Schema", null=True, blank=True)
```

Comment:
`schema` could be a bit too broad name. Let's specify it a bit more, perhaps `beneficiary_schema` or `beneficiary_data_schema` would work better.

Reply:
And please add validation in service or model to ensure that this field is a valid JSON schema.

Reply:
I used TextField according to requirements in the task, but why don't use JSONField?

Reply:
Even if JSONField used I guess json validation should be performed, right?

Reply:
That's a good idea, we can use JSON Field instead of text. You're right that we still need validation, as JSON Field is not same as JSON Schema

Reply:
done